framework/tests/all-systems/InferNullType.java

```diff
@@ -4,8 +4,13 @@
         return input;
     }
 
+    <T> T toInfer2(T input) {
```

Comment:
These test cases caused a crash, which I fixed.  The test cases and fix have been committed.